tools/cldr-code/src/main/java/org/unicode/cldr/test/ConsoleCheckCLDR.java

```diff
@@ -455,30 +432,18 @@ public static void main(String[] args) throws IOException {
         Factory cldrFactory =
                 SimpleFactory.make(sourceDirectories, factoryFilter)
                         .setSupplementalDirectory(new File(CLDRPaths.SUPPLEMENTAL_DIRECTORY));
-        CompoundCheckCLDR checkCldr = CheckCLDR.getCheckAll(cldrFactory, checkFilter);
-        if (checkCldr.getFilteredTestList().size() == 0) {
-            throw new IllegalArgumentException("The filter doesn't match any tests.");
-        }
-        System.out.println("filtered tests: " + checkCldr.getFilteredTests());
+        TestCache testCache = new TestCache(); // for parallelism
+        testCache.setFactory(cldrFactory, checkFilter);
```

Comment:
this is the core change: use TestCache (same as ST! ) to manage the CheckCLDR instances.

This does have an added benefit that we're running more similar code in ConsoleCheck and ST… I don't anticipate divergence otherwise, but it does mean the environ is similar.